src/components/BusinessViews/ImagesCarousel.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from "react";
+import {
+  Image,
+  StyleSheet,
+  Text,
+  View,
+  ScrollView,
+  TouchableHighlight,
+  Modal,
+  Pressable,
+  Alert,
+} from "react-native";
+import ImageView from "react-native-image-viewing";
+
+const Image1 =
+  "https://cdn.vox-cdn.com/thumbor/VAkim2EiaKiIq4pUi295wH99Ces=/0x0:1100x729/1200x800/filters:focal(341x230:517x406)/cdn.vox-cdn.com/uploads/chorus_image/image/67717391/STARBUCKS.0.jpeg";
+const Image2 =
+  "https://content-prod-live.cert.starbucks.com/binary/v2/asset/137-42658.jpg";
+const Image3 =
+  "https://cdn.vox-cdn.com/thumbor/1WVT8VSapMXbHPvsTEPxZp2gUrk=/0x0:1347x897/1200x800/filters:focal(567x342:781x556)/cdn.vox-cdn.com/uploads/chorus_image/image/62192379/starbucksredcups2015.1541431580.jpg";
+const Image4 =
+  "https://cdn.vox-cdn.com/thumbor/bSoWzne8VZvz8tavhebsL7DNik0=/0x0:5860x4008/1200x800/filters:focal(3243x1967:4179x2903)/cdn.vox-cdn.com/uploads/chorus_image/image/67132574/shutterstock_1410002591.0.jpg";
+
+const images = [
+  {
+    uri: Image1,
+  },
+  {
+    uri: Image2,
+  },
+  {
+    uri: Image3,
+  },
+  {
+    uri: Image4,
+  },
+];
+
+const ImagesCarousel = () => {
```

Comment:
It would be good if this component took an array of images as one of its props.

Reply:
I agree. I was waiting for the everyone's different work to be amalgamated first though.

Reply:
I don't think my businessview changes will cause you any issue with making this change, as long as you just change the props for this view and give it a static input/parameters where it's used for now?